src/Prefs.cpp

```diff
@@ -71,20 +70,39 @@ std::unique_ptr<FileConfig> ugPrefs {};
 FileConfig *gPrefs = nullptr;
 int gMenusDirty = 0;
 
-wxDEFINE_EVENT(EVT_PREFS_UPDATE, wxCommandEvent);
+struct MyEvent : wxEvent
+{
+public:
+   explicit MyEvent(int id) : mId{id} {}
+   virtual wxEvent *Clone() const override { return new MyEvent{mId}; }
+   int mId;
+};
+
+wxDECLARE_EVENT(EVT_PREFS_UPDATE, MyEvent);
+wxDEFINE_EVENT(EVT_PREFS_UPDATE, MyEvent);
 
 struct PrefsListener::Impl : wxEvtHandler
 {
    Impl( PrefsListener &owner );
    ~Impl();
-   void OnEvent(wxCommandEvent&);
+   void OnEvent(wxEvent&);
    PrefsListener &mOwner;
 };
 
+static wxEvtHandler hub;
```

Comment:
Why you using `evtHandler` as standalone static variable and not add handler into `PrefsListener` or using `FilterEvents` for handling? I don't have the deep knowleges about `wxEvents`, because little bit confused.